modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/assign_scopes_tab2.jspf

```diff
@@ -19,13 +19,16 @@
 	<%
 	for (Map.Entry<AuthorizationModel, AssignScopesModel.Relations> scopeAliasAuthorizationModelEntry : assignScopesModel.getGlobalAuthorizationModelsRelations().entrySet()) {
 		String scopeAlias = scopeAliasAuthorizationModelEntry.getValue().getScopeAlias();
-		%>
+	%>
 
 		<div class="panel panel-secondary" id="<portlet:namespace /><%= scopeAlias %>">
 			<div class="collapse-icon panel-header panel-header-link" id="<portlet:namespace />globalAccordionHeading<%= scopeAlias %>">
 				<div class="custom-checkbox custom-control" style="display: inline-block;">
+
+					<%--We should be using aui:input instead of this label and input tags --%>
```

Comment:
We tried. Then it doesn't use the new clay design and checkboxes render ugly, are not the new clay checkboxes. Is that expected?

Reply:
Yeah, I guess that in this case we prefer all of them to be consistent so we can easily replace them all at some point with the new markup or components.

Reply:
Makes sense, thanks!